apps/web/components/integrations/IntegrationListItem.tsx

```diff
@@ -15,7 +14,7 @@ function IntegrationListItem(props: {
   return (
     <ListItem expanded={!!props.children} className={classNames("flex-col")}>
       <div className={classNames("flex w-full flex-1 items-center space-x-2 p-3 rtl:space-x-reverse")}>
-        {props.imageSrc && <Image width={40} height={40} src={`/${props.imageSrc}`} alt={props.title} />}
+        {props.imageSrc && <img className="h-10 w-10" src={props.imageSrc} alt={props.title} />}
```

Comment:
As we have some apps with remote url logo if we use Next/Image we need to add those domains on our next.config.js under images: ['cdn.to-remote-logo.com'], or just use <img/> for this case